controls/SV-204392.rb

```diff
@@ -34,21 +32,21 @@
 
     Reset the permissions of files within a package with the following command:
 
-    #rpm --setperms <packagename>
-  "
+    #rpm --setperms <packagename>'
   impact 0.7
-  tag 'legacy': ['V-71849', 'SV-86473']
-  tag 'severity': 'high'
-  tag 'gtitle': 'SRG-OS-000257-GPOS-00098'
-  tag 'satisfies': ['SRG-OS-000257-GPOS-00098', 'SRG-OS-000278-GPOS-00108']
-  tag 'gid': 'V-204392'
-  tag 'rid': 'SV-204392r646841_rule'
-  tag 'stig_id': 'RHEL-07-010010'
-  tag 'fix_id': 'F-36302r646840_fix'
-  tag 'cci': ['CCI-001494', 'CCI-001496', 'CCI-002165', 'CCI-002235']
+  tag legacy: ['V-71849', 'SV-86473']
+  tag severity: 'high'
+  tag gtitle: 'SRG-OS-000257-GPOS-00098'
+  tag satisfies: ['SRG-OS-000257-GPOS-00098', 'SRG-OS-000278-GPOS-00108']
+  tag gid: 'V-204392'
+  tag rid: 'SV-204392r646841_rule'
+  tag stig_id: 'RHEL-07-010010'
+  tag fix_id: 'F-36302r646840_fix'
+  tag cci: ['CCI-001494', 'CCI-001496', 'CCI-002165', 'CCI-002235']
   tag nist: ['AU-9', 'AU-9 (3)', 'AC-3 (4)', 'AC-6 (10)']
-  tag subsystems: ["permissions","package","rpm"]
-  tag 'host', 'container'
+  tag subsystems: ['permissions', 'package', 'rpm']
+  tag host: nil
+  tag container: nil
```

Comment:
The tags without values can be left alone without needing a value to be specified. It's valid syntax based on the [InSpec docs](https://docs.chef.io/inspec/dsl_inspec/#syntax)

Reply:
The quotes should also be kept around the keys for `host` and `container`